README.md

```diff
@@ -28,11 +28,13 @@ You need:
 
 * Patience, to wait for [Jimp](https://github.com/oliver-moran/jimp) to convert your images; or [Imagemagick](http://www.imagemagick.org/script/index.php), if you want much better speed and full animated GIF support; OR, on Macs, the very fast [imagecrunch](http://github.com/punkave/imagecrunch) utility. You can also write a backend for something else (look at `imagemagick.js`, `imagecrunch.js`, and `jimp.js` for examples; just supply an object with the same methods, you don't have to supply a factory function).
 
+* If you want GIF support: [Imagemagick](http://www.imagemagick.org/script/index.php) or (Mac-specific) [imagecrunch](http://github.com/punkave/imagecrunch). `jimp` requires no installation of system packages, but it does not yet support GIF.
```

Comment:
A little confused if this is duplicating some of the information in the preceding paragraph?

Reply:
Yes I will give that text one more pass. What changed is that we realized there's no GIF support *at all* in the default jimp backend you get if you don't have imagemagick installed. Funny that no one complained...